PhysicsTools/SelectorUtils/interface/CutApplicatorWithEventContentBase.h

```diff
@@ -17,6 +18,9 @@
 
 class CutApplicatorWithEventContentBase : public CutApplicatorBase {
  public:  
+
+ CutApplicatorWithEventContentBase(): CutApplicatorBase() {}
```

Comment:
Also only needed for expreval.

So no need in 74X.

-L

(Sent from my Nexus 6)
On Jul 9, 2015 7:58 PM, "Slava Krutelyov" <notifications@github.com> wrote:

> In
> PhysicsTools/SelectorUtils/interface/CutApplicatorWithEventContentBase.h
> <https://github.com/cms-sw/cmssw/pull/9659#discussion_r34283629>:
>
> > @@ -17,6 +18,9 @@
> >
> >  class CutApplicatorWithEventContentBase : public CutApplicatorBase {
> >   public:
> > +
> > + CutApplicatorWithEventContentBase(): CutApplicatorBase() {}
>
> not in 74X
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cms-sw/cmssw/pull/9659/files#r34283629>.
>